.github/workflows/ci.yml

```diff
@@ -243,10 +244,12 @@ jobs:
           - { name: 'Android', target: aarch64-linux-android }
           - { name: 'iOS', target: aarch64-apple-ios }
           - { name: 'Linux', target: x86_64-unknown-linux-gnu }
-          - { name: 'macOS', target: x86_64-apple-darwin }
+          - { name: 'macOS x86_64', target: x86_64-apple-darwin }
+          - { name: 'macOS Aarch64', target: aarch64-apple-darwin }
```

Comment:
I'm not sure it's _really_ necessary for cargo deny, there isn't much of a difference in our codebase between these two

Reply:
Should I remove the X86 then?

Reply:
Yeah, it's probably best to focus on Aarch64